etc/ssl/alt_names.txt

```diff
@@ -0,0 +1 @@
+subjectAltName = DNS:docker.localhost
```

Comment:
Need clarification on this

Reply:
Sorry, this was also supposed to be CANDIG_DOMAIN